modules/documentation/ex_readme.md

```diff
@@ -0,0 +1,8 @@
+# Improve a README
```

Comment:
I guess this still needs some work? Or at least it is not clear for me what I am supposed to do as a learner.